plugins/callback/logger_runtime.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+# (c) 2020 Ashish Varshney <er.ashish.av@gmail.com>
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import collections
+import time
+import logging
+from ansible.plugins.callback import CallbackBase
+
+
+DOCUMENTATION = '''
+    callback: logger_runtime
+    type: aggregate
+    short_description: adds time information to tasks
+    version_added: "2.0"
```

Comment:
```suggestion
    version_added: "2.0"
```
@felixfontein still 2.10?